<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GameScene" name="gameScene" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>721</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>280</width>
      <height>90</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Your Knowledge</string>
    </property>
   </widget>
   <widget class="FishEducation" name="fishEduction" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>981</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameScene</class>
   <extends>QWidget</extends>
   <header>gamescene.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FishEducation</class>
   <extends>QWidget</extends>
   <header>fisheducation.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
